pandas/core/arrays/categorical.py

```diff
@@ -2529,6 +2529,26 @@ def _delegate_method(self, name, *args, **kwargs):
         if res is not None:
             return Series(res, index=self.index, name=self.name)
 
+    @property
+    def categorical(self):
+        warn("s.cat.categorical has been deprecated",
```

Comment:
Say `'Series.cat.categorical'`, rather than `'s'`. Also, say what to do instead. Something like `Use the attributes on 'Series.cat' directly instead.`.